src/org/labkey/targetedms/SkylinePort/Irt/RegressionLine.java

```diff
@@ -72,7 +72,8 @@ private void setIntercept(double value)
 	 @param x Value in x dimension
 	 @return 
 	*/
-	public double GetY(double x)
+	@Override
+    public double GetY(double x)
```

Comment:
Formatting looks mangled. Maybe it was pre-mangled and we should do a general reformat in this PR too?